Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/model/CompiledLibrary.kt

```diff
@@ -0,0 +1,281 @@
+package org.cqframework.cql.cql2elm.model
+
+import java.util.*
+import kotlin.collections.ArrayList
+import kotlin.collections.HashMap
+import org.hl7.cql.model.DataType
+import org.hl7.cql.model.NamespaceManager
+import org.hl7.cql_annotations.r1.Annotation
+import org.hl7.elm.r1.CodeDef
+import org.hl7.elm.r1.CodeSystemDef
+import org.hl7.elm.r1.ConceptDef
+import org.hl7.elm.r1.Element
+import org.hl7.elm.r1.ExpressionDef
+import org.hl7.elm.r1.FunctionDef
+import org.hl7.elm.r1.IncludeDef
+import org.hl7.elm.r1.Library
+import org.hl7.elm.r1.ParameterDef
+import org.hl7.elm.r1.UsingDef
+import org.hl7.elm.r1.ValueSetDef
+import org.hl7.elm.r1.VersionedIdentifier
+
+@Suppress("TooManyFunctions")
+class CompiledLibrary {
+    var identifier: VersionedIdentifier? = null
```

Comment:
I really like Kotlin's clarity about what's nullable and what isn't.  It would be nice if such a mechanism existed in Java beyond Optional and @Nullable we could use.